console/algorithms/Cargo.toml

```diff
@@ -14,13 +14,11 @@ harness = false
 [dependencies.snarkvm-console-types]
 path = "../types"
 version = "0.9.13"
-default-features = false
 features = [ "field", "group", "integers", "scalar" ]
 
 [dependencies.snarkvm-fields]
 path = "../../fields"
 version = "0.9.13"
-default-features = false
```

Comment:
`fields` `default-features` should be left alone (except for the parallel flags)